### DISCUSSION TITLE
Checkboxes in the issue template get shown as "tasks"

### DISCUSSION THREAD

**Discussion by bennuttall** [community/community#69471] [Projects and Issues]
### Select Topic Area

Product Feedback

### Feature Area

Issues

### Body

I have an issue template which involves asking the user some questions, like which versions their issue relates to, and for some questions I use checkboxes:

![Screenshot from 2023-10-07 12-29-24](https://github.com/community/community/assets/1609725/dc5faf72-656f-4fa3-a3c0-0c5992765724)

It works fine but the checkboxes get counted as "tasks" which isn't really right:

![Screenshot from 2023-10-07 12-30-25](https://github.com/community/community/assets/1609725/2ddd98b4-4ffd-47eb-8451-06ed244ccdea)

Example: https://github.com/piwheels/packages/issues/394

### Guidelines

- [X] I have read the above statement and can confirm my post is relevant to the GitHub feature areas [Issues](https://docs.github.com/en/issues/tracking-your-work-with-issues/about-issues) and/or [Projects](https://docs.github.com/en/issues/planning-and-tracking-with-projects/learning-about-projects/about-projects).

**Comment by himanshu-03** [comment]
Hey @bennuttall !!
You are absolutely right about the issues and the feedbacks.

However, if you notice GitHub's documentation, there they have clearly explained the actual use of the task lists. Turns out most of the people are using this wrong and are misinterpreated with tasks lists being the multi choice options.

You can refer their documentation here: https://docs.github.com/en/get-started/writing-on-github/working-with-advanced-formatting/about-task-lists

**Reply by bennuttall** [reply]
I don't understand what you're saying. I understand the purpose of task lists. I just don't want my checkboxes to be treated as tasks in this case.

**Comment by t-hamano** [comment]
I am running into the same problem.

[The WordPress Gutenberg project](https://github.com/WordPress/gutenberg) I am working on requires checkboxes in the issue template for the reporter to confirm when submitting an issue. These checkboxes are _only_ required when submitting an issue, they are not tasks:

![image](https://github.com/user-attachments/assets/5ce9e0c9-89bb-48e9-adcc-42cac9eee085)

Nevertheless, when you submit an issue they are treated as tasks and show "2 tasks done":

![image](https://github.com/user-attachments/assets/be20fb16-914a-45bd-8553-fe71ad94a1a7)

I would like to see an option added to not treat the checkboxes as tasks in the issue template, for example:

```yaml
name: Bug report
body:
    - type: checkboxes
      id: existing
      attributes:
          label: Please confirm that you have searched existing issues in the repo.
          options:
              - label: 'Yes'
                required: true
          is_task: false
```
